crates/rpc/src/error.rs

```diff
@@ -99,6 +99,12 @@ pub enum ApplicationError {
     StorageProofNotSupported,
     #[error("Proof is missing")]
     ProofMissing,
+    #[error("Invalid subscription id")]
+    InvalidSubscriptionID,
+    #[error("Too many addresses in filter sender_address filter")]
```

Comment:
Doesn't make a whole lot of sense to me... Is the text coming from some other document?

Reply:
Text is straight from Starknet spec:

https://github.com/starkware-libs/starknet-specs/blob/v0.8.0/api/starknet_ws_api.json#L371